common/models/user.js

```diff
@@ -560,19 +560,26 @@ module.exports = function(User) {
         err.code = 'EMAIL_NOT_FOUND';
         return cb(err);
       }
-      // create a short lived access token for temp login to change password
-      // TODO(ritch) - eventually this should only allow password change
-      user.accessTokens.create({ ttl: ttl }, function(err, accessToken) {
-        if (err) {
-          return cb(err);
-        }
-        cb();
+      if (user && user.emailVerified) {
         UserModel.emit('resetPasswordRequest', {
           email: options.email,
-          accessToken: accessToken,
```

Comment:
How are you going to construct an authenticated URL when there is no accessToken available?